ipi_tests/reg_test/runstools.py

```diff
@@ -1,46 +0,0 @@
-import subprocess as sp
-from pathlib import Path
-import pytest
-import time
-import os
-import shutil
-import tempfile
-
-
-class Runner(object):
-    def __init__(self, parent):
-        self.parent = parent
-
-    def _run(self, cmd1, cmd2, cwd):
```

Comment:
Same for all the functions that you have defined. Can you add some description using docstrings ?